components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -24,6 +24,11 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
   reportLoading = false;
   RFC2822 = DateTime.RFC2822;
   returnParams: ReturnParams = {};
+  page$: Subject<number> = new Subject<number>();
+  page = 1;
```

Comment:
I had to add the "page" field because when first opening up the page picker the first page was not highlighted.